ext/intl/calendar/calendar_methods.cpp

```diff
@@ -43,6 +43,27 @@ extern "C" {
 
 using icu::Locale;
 
+#define ZEND_VALUE_ERROR_INVALID_FIELD(argument, zpp_arg_position)							\
+	if (argument < 0 || argument >= UCAL_FIELD_COUNT) {										\
+		zend_argument_value_error(getThis() ? ((zpp_arg_position)-1) : zpp_arg_position,	\
+			"must be a valid field");														\
+		RETURN_THROWS();																	\
+	}
+
+#define ZEND_VALUE_ERROR_OUT_OF_BOUND_VALUE(argument, zpp_arg_position)						\
+	if (argument < INT32_MIN || argument > INT32_MAX) {										\
+		zend_argument_value_error(getThis() ? ((zpp_arg_position)-1) : zpp_arg_position,	\
+			"must be between %d and %d", INT32_MIN, INT32_MAX);								\
+		RETURN_THROWS();																	\
+	}
+
+#define ZEND_VALUE_ERROR_INVALID_DAY_OF_WEEK(argument, zpp_arg_position)					\
+	if (argument < UCAL_SUNDAY || argument > UCAL_SATURDAY) {								\
+		zend_argument_value_error(getThis() ? ((zpp_arg_position)-1) : zpp_arg_position,	\
+			"must be a valid day of the week");												\
+		RETURN_THROWS();																	\
+	}
```

Comment:
Please don't indent the `\`. Just put it at the end of the line after a single space, no alignment.